src/plugin/helpers/prosemirror.ts

```diff
@@ -13,13 +13,21 @@ const nodesBetween = (state: EditorState, _from: number, _to: number) => {
   const range = [_from, _to];
   range.sort((a, b) => a - b);
   const [from, to] = range;
-
   const arr: NodesBetweenArgs[] = [];
-  state.doc.nodesBetween(
-    from,
-    to,
-    (node, pos, parent, index) => !!arr.push([node, pos, parent, index])
```

Comment:
The !! in `!!arr.push([node, pos, parent, index])` was to achieve a boolean typing, I'm not sure we were really using the logic in practice. See 15e19d1.